joy_node:
  ros__parameters:
    device_id: 0
    deadzone: 0.05
    autorepeat_rate: 20.0

jtc_node:
  ros__parameters:
    axis_linear:
      x:
        forward: 5
        backward: 2
    scale_linear:
      x: 0.5
    
    axis_angular:
      yaw: 0
    scale_angular:
      yaw: 2.0